test/Tests/controlled_invariant.jl

```diff
@@ -107,7 +103,7 @@ function ci_piecewise_semiell_mci_homogeneous_test(optimizer, config)
     ci_square_test(
         optimizer, config, true,
         Ellipsoid(symmetric=true, piecewise=polar_mci),
-        set -> L1_heuristic(set), 0.9349838949990186,
+        set -> L1_heuristic(set), 2.9909434642487316,
```

Comment:
This was incorrectly modified in https://github.com/blegat/SetProg.jl/pull/16
The integral was wrong because the `Qi` were not assigned in the right ordered that changed for some reason